addon/controllers/building/edit/form.js

```diff
@@ -88,16 +88,21 @@ export default class BuildingFormController extends Controller {
         this.buildingWork
       );
 
-      const project = yield this.constructionProject.getFromCacheOrApi(
-        this.model.projectId
+      // refresh work list
+      const project = yield this.constructionProject.get(this.model.projectId);
+
+      // remove default work
+      yield Promise.all(
+        project.work.map((work) =>
+          work.building.isNew
```

Comment:
```suggestion
          // Since all works have per default a new building model, we can figure out the default work by filtering `building.isNew`.
          work.building.isNew
```